includes/polling/os/ceraos.inc.php

```diff
@@ -1,5 +1,31 @@
 <?php
+/*
+ * LibreNMS
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
 
-$hardware = snmp_get($device, 'genEquipInventoryCardName.127', '-Osqnv', 'MWRM-UNIT-MIB');
-$serial = snmp_get($device, 'genEquipInventorySerialNumber.127', '-Osqnv', 'MWRM-UNIT-MIB');
-$version = snmp_get($device, 'genEquipMngSwIDUVersionsRunningVersion.1', '-Osqnv', 'MWRM-UNIT-MIB');
+$ceragon_type = snmp_get($device, 'sysObjectID.0', '-mSNMPv2-MIB -Oqv', '');
+$hardware = rewrite_ceraos_hardware($ceragon_type); // function in ./includes/rewrites.php
+$version = snmp_get($device, 'genEquipMngSwIDUVersionsRunningVersion.1', '-mMWRM-RADIO-MIB -Oqv', '');
+$serial = snmp_walk($device, 'genEquipInventorySerialNumber', '-mMWRM-RADIO-MIB -Oqv', '');
```

Comment:
Just to point out that snmp_get_multi() is a disaster!

It was obviously written for a specific use case and does not easily support any other use.
I am getting $version, $lat, $long as multi_get. $sersion is index 1, lat and log are index 0. result is:

[1] => [OID for version] = version
[0] => [OID for lat] = lat
          [OID for long] = long

and all because someone decided to munge their data in a generic function...